src/d3a/models/area/__init__.py

```diff
@@ -414,13 +419,18 @@ def _update_myco_matcher(self) -> None:
         """Update the markets cache that the myco matcher will request"""
         bid_offer_matcher.update_area_uuid_markets_mapping(
             area_uuid_markets_mapping={
-                self.uuid: {"markets": self.all_markets, "current_time": self.now}})
+                self.uuid: {"markets": self.all_markets,
+                            "settlement_markets": list(self.settlement_markets.values()),
```

Comment:
Is there a specific reason to use `list` on this? The following should work as well:
```suggestion
                            "settlement_markets": self.settlement_markets.values(),
```

Reply:
Should work fine even without the casting to list, removing